controllers/controller_suite_test.go

```diff
@@ -74,40 +77,38 @@ func startTestEnv() {
 	Expect(err).NotTo(HaveOccurred())
 
 	//+kubebuilder:scaffold:scheme
+	k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{Scheme: scheme.Scheme})
+	Expect(err).NotTo(HaveOccurred())
+	Expect(k8sManager).NotTo(BeNil())
 
 	k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})
 	Expect(err).NotTo(HaveOccurred())
 	Expect(k8sClient).NotTo(BeNil())
 
-	k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{
-		Scheme:  scheme.Scheme,
-		Metrics: metricsServer.Options{BindAddress: "0"},
-	})
-	Expect(err).ToNot(HaveOccurred())
+	mockManager, _ := lease.NewManager(k8sClient, "")
```

Comment:
Nit: there are a lot of managers involved, maybe `mockLeaseManager` is better name ?

Reply:
`mockLeaseManager` is already used in [nodemaintenance_controller_test.go](https://github.com/medik8s/node-maintenance-operator/blob/main/controllers/nodemaintenance_controller_test.go#L342-L348). 
Maybe `tempMockManager`?